certbot-apache/certbot_apache/interfaces.py

```diff
@@ -272,6 +318,7 @@ class BlockNode(ParserNode):
 
     The applicable parameters are dependent on the underlying configuration language
     and its grammar.
+
```

Comment:
Looks like we need a description of the `children` here. We deleted the property below but didn't document it here.